big_transfer.py

```diff
@@ -20,78 +20,146 @@ class BigTransferEncoder(Executor):
     Internally, :class:`BigTransferEncoder` wraps the models from
     https://storage.googleapis.com/bit_models/.
 
-    .. warning::
-
-        Known issue: this does not work on tensorflow==2.2.0,
-        https://github.com/tensorflow/tensorflow/issues/38571
-
     :param model_path: the path of the model in the `SavedModel` format.
         The pretrained model can be downloaded at
         wget https://storage.googleapis.com/bit_models/Imagenet21k/[model_name]/feature_vectors/saved_model.pb
         wget https://storage.googleapis.com/bit_models/Imagenet21k/[model_name]/feature_vectors/variables/variables.data-00000-of-00001
         wget https://storage.googleapis.com/bit_models/Imagenet21k/[model_name]/feature_vectors/variables/variables.index
 
-        ``[model_name]`` includes `R50x1`, `R101x1`, `R50x3`, `R101x3`, `R152x4`
+    :param model_name: includes `R50x1`, `R101x1`, `R50x3`, `R101x3`, `R152x4`
+
+    This encoder checks if the specified model_path exists.
+    If it does exist, the model in this folder is used.
+    If it does not exist, the model specified in the model_name will be
+    downloaded into this path and the downloaded model is used.
+
+    In the end, the `model_path` should be a directory path,
+    which has the following structure:
 
-        The `model_path` should be a directory path, which has the following structure.
+    .. highlight:: bash
+     .. code-block:: bash
+        .
+        ├── saved_model.pb
+        └── variables
+            ├── variables.data-00000-of-00001
+            └── variables.index
 
-        .. highlight:: bash
-         .. code-block:: bash
+    :param channel_axis: the axis id of the channel, -1 indicate the color
+        channel info at the last axis. If given other, then `
+        `np.moveaxis(data, channel_axis, -1)`` is performed before :meth:`encode`.
 
-            .
-            ├── saved_model.pb
-            └── variables
-                ├── variables.data-00000-of-00001
-                └── variables.index
+    :param default_traversal_paths: Traversal path through the docs
+    :param default_batch_size: Batch size to be used in the encoder model
 
-        :param channel_axis: the axis id of the channel, -1 indicate the color
-            channel info at the last axis. If given other, then `
-            `np.moveaxis(data, channel_axis, -1)`` is performed before :meth:`encode`.
+    :param: on_gpu: If true, the GPU will be used. Make sure to have
+        tensorflow-gpu==2.5 installed
     """
+    DEFAULT_TRAVERSAL_PATHS = ['r']
```

Comment:
I think we can put this directly as default param?
`default_traversal_paths: List[str] =['r'],`